packages/config/src/lib/config.ts

```diff
@@ -102,6 +104,8 @@ export type ConfigType = {
     ignorePaths?: string[];
     env?: string[];
   };
+  RCT_USE_RN_DEP?: number,
+  RCT_USE_PREBUILT_RNCORE?: number,
```

Comment:
realistically we usually want to keep these 2 synchronized, right? Let's use `usePrebuiltRNCore` as a name and set both env vars accordingly in sync